lesson2/src/main/java/by/tms/lesson2/servlets/RegisterServlet.java

```diff
@@ -0,0 +1,59 @@
+package by.tms.lesson2.servlets;
+
+import by.tms.lesson2.classes.User;
+import by.tms.lesson2.classes.UserBase;
+import by.tms.lesson2.exceptions.UserException;
+import by.tms.lesson2.xml.ParserXML;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Map;
+
+@WebServlet(name = "RegisterServlet", urlPatterns = "/register")
+public class RegisterServlet extends HttpServlet {
+
+    @Override
+    public void init() throws ServletException {
+        super.init();
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        processRequest(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        processRequest(req, resp);
+    }
+
+    private void processRequest(HttpServletRequest req, HttpServletResponse resp) throws IOException, ServletException {
+        resp.setCharacterEncoding("UTF-8");
+
+        String username = req.getParameter("username");
+        String name = req.getParameter("name");
+        String password = req.getParameter("password");
+
```

Comment:
username, name and password could be blank ("") - you should to prevent saving user with empty fields